modules/blocks/verify.js

```diff
@@ -573,22 +573,41 @@ Verify.prototype.deleteBlockProperties = function(block) {
 	}
 	if (reducedBlock.totalAmount === 0) {
 		delete reducedBlock.totalAmount;
+	} else {
+		reducedBlock.totalAmount = new bignum(reducedBlock.totalAmount);
```

Comment:
There is something wrong with logic here. Before if property was `0` we removed it, now we create bignum instance if it's not `0`. We should always create instance or never create.